packages/terra-clinical-item-view/docs/README.md

```diff
@@ -0,0 +1,52 @@
+# Terra Clinical Item View
+
+The Terra ClinicalItemView component allows displays to be organized into rows and column and themed, while providing means to add accessory elements.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-clinical-item-view`
+  - `yarn add terra-clinical-item-view`
+
+## React Usage
+
+The React component can be initialized like so:
+```
+import ClinicalItemView from 'ClinicalItemView;
+
+<ClinicalItemView
+  layout="oneColumn"
+  textEmphasis="default"
+  isTruncated=true
+  accessoryAlignment="top"
+  leftAccesory=<img alt="Graphic" />
+  displays={[<ClinicalItemView.Display text="display 1" />, <ClinicalItemView.Display text="display 2" />]}
+  comment=<ClinicalItemView.Comment text="comment" />
+/>
+
+<Display
+  text="display text"
+  textStyle="attention"
+  isTruncated=true
+  icon=<img alt="Graphic" />
+/>
+
+<Comment
+  text="display text"
+  textStyle="attention"
+  isTruncated=true
+/>
+```
+
+## React Props
+
+| Prop | Type | Default | Description|
+|------|------|---------|------------|
+|`layout`|String|`oneColumn`|The number of columns used to layout display. One of `oneColumn` or `twoColumns`.|
```

Comment:
For the time being we have leave this, in case we want to add layouts that aren't column style, verticle etc.